Manual.md

```diff
@@ -132,6 +135,83 @@ the Void packages system.
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
+<a id="cross_compilation"></a>
+### Cross compilation
+
+Void Linux places high emphasis on having packages being able to be
+cross-compiled.
```

Comment:
`Void Linux places high emphasis on being able to cross compile the packages in its repository` maybe?